contracts/CodexTitle.sol

```diff
@@ -87,20 +100,25 @@ contract CodexTitle is ERC721Token {
     tokenURIPrefix = _tokenURIPrefix;
   }
 
-  function mint(
```

Comment:
Why was this removed? Isn't this the "shorthand" way of calling `mint()` without having to specify the optional parameters?

Reply:
This is related to the `msg.sender` stuff I was talking about before. If we have cascading functions like these, we also have to pass msg.sender along which can get messy pretty fast. I opted to remove this override and instead rely on the caller to pass in empty string for those values if needed.

Reply:
Fair enough. There don't seem to be any places we use the shorthand right now so I think it's fine.